myosuite/envs/myo/assets/leg/myoosl_control.py

```diff
@@ -11,6 +11,7 @@ def __init__(self,
                  body_mass,
                  init_state='e_stance',
                  hardware_param=None,
+                 n_sets=4,
```

Comment:
can you add docstrings indicating what is this parameter about?

Reply:
Yes, added in the doc string